<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 222, in render
    yaml_text = _get_template(meta_yaml, selector_dict).render(render_dict)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 152, in _get_template
    return _jinja_silent_undef.from_string(meta_yaml_selectors_applied)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/jinja2/environment.py", line 1105, in from_string
    return cls.from_code(self, self.compile(source), gs, None)
                               ^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<unknown>", line 30, in template
jinja2.exceptions.TemplateSyntaxError: unexpected '}'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 734, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 244, in from_file
    raise exc
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 235, in from_file
    recipe._load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 167, in _load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 284, in render
    self.meta = renderer.render(self.recipe_dir, self.dump(), self.selector_dict, self.renderer)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 224, in render
    raise JinjaRenderFailure(recipe_dir, message=exc.message, line=exc.lineno)
percy.render.exceptions.JinjaRenderFailure: (PosixPath('/tmp/abs_8fmp0vr0xy/clone/recipe'), "unexpected '}' (at line 30)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 126, in main
    result = linter.lint(recipes, subdir, args.variant_config_files, args.exclusive_config_files, args.fix)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 634, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 751, in lint_file
    recipe = _recipe.Recipe(recipe_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 106, in __init__
    self.recipe_dir = recipe_file.parent
                      ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'parent'

ERROR conda.cli.main_run:execute(49): `conda run conda-lint /tmp/abs_8fmp0vr0xy/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -42,6 +42,7 @@ test:
     - pip
     - pytest
     - pandas
+    - pandas-stubs
     - black
     - feedparser
     - isort

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -26,6 +26,8 @@ requirements:
 
 # skipping because not building latest release
 {% set tests_to_skip = "test_pandas_stubs_fork_is_synchronized or test_version_bump" %}
+# skip wheel tests since we are relying on conda-build
+{% set tests_to_skip = tests_to_skip + " or test_wheel_is_built_correctly or test_wheel_can_be_installed" %}
 
 test:
   source_files:
@@ -34,7 +36,7 @@ test:
     - nptyping
   commands:
     - pip check
-    # ignoring test with missing packages
+    # ignoring test file with missing packages
     - pytest --ignore=tests/test_beartype.py --ignore=tests/test_pyright.py -k "not ({{ tests_to_skip }})" tests
   requires:
     - pip

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -28,6 +28,9 @@ requirements:
 {% set tests_to_skip = "test_pandas_stubs_fork_is_synchronized or test_version_bump" %}
 # skip wheel tests since we are relying on conda-build
 {% set tests_to_skip = tests_to_skip + " or test_wheel_is_built_correctly or test_wheel_can_be_installed" %}
+# skip tests where type argument is necessary (only working on py311)
+{% set tests_to_skip = tests_to_skip + " or test_mypy_accepts_dataframe or test_mypy_disapproves_dataframe_with_wrong_function_arguments" %}  # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_mypy_knows_of_dataframe_methods or test_basic_nptyping_code" %}                              # [py<311]
 
 test:
   source_files:

